rules/misc_rules.build_defs

```diff
@@ -451,7 +452,7 @@ def remote_file(name:str, url:str|list, hashes:list=None, out:str=None, binary:b
     urls = [url] if isinstance(url, str) else url
     url = urls[0]
 
-    urls_str = ' '.join(url) if isinstance(url, list) else url
+    urls_str = ' '.join(urls)
```

Comment:
I changed this line because it seems to conflict with the preceding line and it broke the multi-url feature